also install wannier2pw.x utility (not just build it)
see also https://gitlab.com/QEF/q-e/-/merge_requests/2916
author: Kenneth Hoste (HPC-UGent)
--- q-e-qe-7.5/PP/CMakeLists.txt.orig	2025-08-19 11:57:12.000000000 +0200
+++ q-e-qe-7.5/PP/CMakeLists.txt	2026-08-04 20:19:37.313463103 +0200
@@ -605,6 +605,7 @@
     qe_pp_pmw_exe
     qe_pp_projwfc_exe
     qe_pp_pw2wannier90_exe
+    qe_pp_wannier2pw_exe
     qe_pp_pw2critic_exe
     qe_pp_wfck2r_exe
     qe_pp_initial_state_exe
